build/src/server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const {spawn} = require('child_process');
+const app = express();
+const port = 8000;
+app.use(bodyParser.json());
+app.use(bodyParser.text());
+
+app.get('/address/:address', (req, res) => {
+    
+    if (req.params.address.length != 42)
+        return res.send({status: "err", message: "Expecting an Ethereum address 42 characters long."});
+  
+    let acctScrape = spawn("acctScrape", ['--list', req.params.address]);
+
+    acctScrape.stdout.on('data', (chunk) => {
+        console.log(`data being written: ${chunk}`)
+        return res.write(chunk);
+    })
+    acctScrape.stderr.on('data', (err) => {
+        console.log(`error being written: ${err}`);
+        return res.write(err);
+    })
```

Comment:
It's nice to see this in concrete code. This is what I've been envisioning for a very long time. I love that it's so minimal. It should be minimal.